docs: Updates to required ports (#19755)
hashicorp/consul: website/content/docs/install/ports.mdx

We recommend using gRPC TLS instead.

### Client gRPC TLS

The following table lists information about the Consul client's gRPC with TLS port defaults:

| Default port | Protocol | Default status       |
| :----------- | :------- | :------------------- |
| `8503`       | TCP      | Disabled by default  |

This port receives incoming traffic from the gateways and Envoy proxies registered to this client. We recommend using `8503` as your conventional gRPC port number, as it allows some tools to work automatically.

This port is disabled by default. You can enable it in the [agent configuration file](/consul/docs/agent/config/config-files#ports) or using the [`consul agent` CLI command](/consul/docs/agent/config/cli-flags).

### Client LAN serf

The following table lists information about the Consul client's LAN serf port defaults:

| Default port | Protocol    | Default status     |
| :----------- | :---------- | :----------------- |
| `8301`       | TCP and UDP | Enabled by default |

This port sends and receives traffic from Consul clients and Consul servers in the same datacenter. Refer to [gossip protocol](/consul/docs/architecture/gossip) for more information.

> When running Enterprise deployments that use network segments or multiple admin partitions, all Consul clients across still require access to this port across all segments or partitions.

Reviewer: ```suggestion
When running Enterprise deployments that use network segments or admin partitions, Consul clients _within_ a segment or partition require access to each other's ports but not _across_ segments/partitions.
```
